### DISCUSSION TITLE
How to alter footer note on every page in the website?

### DISCUSSION THREAD

**Discussion by Power-tile** [requarks/wiki#2951] [Help / Questions] [answered]
On [the Wiki.js documentation](https://docs.requarks.io/), for every page, there is a footer that can be viewed by scrolling to the bottom of each page:

![image](https://user-images.githubusercontent.com/44731563/105638266-b50f5980-5eac-11eb-955f-d89fe2a3c641.png)

How can I quickly change this information to show desired information? Is there a feature built-in in the admin settings?

**Comment by NGPixel** [comment, marked as answer]
Under **Adminstration** > **General**
